src/js/structs/Job.js

```diff
@@ -12,6 +12,13 @@ module.exports = class Job extends Item {
     return cmd;
   }
 
+  getCpus() {
+    // Default to the minimum number of cpus per Mesos offer
```

Comment:
Yeah, thats a really good idea! But as you guessed not in the scope of this PR. I'll create a tech debt issue.